Changelog.md

```diff
@@ -29,6 +29,7 @@ Compiler Features:
 
 Bugfixes:
  * AST Import: For constructors, a public visibility is ignored during importing.
+ * References Resolver: Detect circular references to other contracts across libraries and free functions
```

Comment:
needs to be moved

Reply:
due to rebasing I didn't see the new release entry and thought you meant it should be moved to bugfixes (previously new features).